ethereum/paraswap/parsers.json

```diff
@@ -184,12 +184,12 @@
                     },
                     "screen": [
                         {
-                            "label": "Received minimum amount",
-                            "name": "toAmount"
+                            "label": "Amount to send",
+                            "name": "fromAmount"
                         },
                         {
-                            "label": "Sent amount",
-                            "name": "fromAmount"
+                            "label": "Minimum amount to be received",
```

Comment:
This does not match what we have on the figma right?
https://www.figma.com/file/BK1aCdRYEoRrfcmgtH5Xd1/Smart-Contract-Interactions-%26-dApp-Browser?node-id=646%3A7231